lib/class-wp-theme-json.php

```diff
@@ -50,6 +50,7 @@ class WP_Theme_JSON {
 		'--wp--style--color--link',
 		'background',
 		'backgroundColor',
+		'border',
```

Comment:
The values on this array need to be the name of the property as they appear in the `PROPERTIES_METADATA`, so this should have been `borderRadius`. There's a couple of other things I'd like to share:

- these styles are attached to the `:root` element, so I'm not sure borderRadius makes a lot of sense here
- the global styles sidebar doesn't have a corresponding UI control for the user to tweak this property ― neither for the global or the group block.

In https://github.com/WordPress/gutenberg/pull/28320 I'm removing the support for `borderRadius` for the root element but could also just rename the property to fix the issue.

@aaronrobertshaw would you be available to create a follow-up PR that adds the radius control to the global styles sidebar if the block supports it?

Reply:
@nosolosw thank you for the explanation. I agree, a `border-radius` value on `:root` doesn't make a lot of sense. 

It was suggested via feedback on other border radius related PRs that there should be a "global" border radius value that themes could use or supply to achieve a more consistent visual result. I mention this as having the border radius as a CSS variable on the `:root` selector would help achieve this. 

In https://github.com/WordPress/gutenberg/pull/27991 I've used a similar approach adding a CSS variable for the border radius so that inner elements could adjust the value to keep uniform spacing between their border and their parent's.

It would be great to get your thoughts on that PR if you can spare the time.

> @aaronrobertshaw would you be available to create a follow-up PR that adds the radius control to the global styles sidebar if the block supports it?

I can do however I had been advised to avoid adding any further sidebar controls until https://github.com/WordPress/gutenberg/issues/27331 lands. Complicating matters and the UI, there are also requests to include support for other border properties such as width and style. I realise these aren't in place at the moment, just mentioning as they seemed to have contributed to the decision to hold off adding extra controls.

Again, I'm happy to do the follow-up if you can confirm I'm ok to do so.

Reply:
I've left my comments on https://github.com/WordPress/gutenberg/pull/27991 and https://github.com/WordPress/gutenberg/pull/27664#issuecomment-762909628

> I can do however I had been advised to avoid adding any further sidebar controls until #27331 lands. Complicating matters and the UI, there are also requests to include support for other border properties such as width and style. I realise these aren't in place at the moment, just mentioning as they seemed to have contributed to the decision to hold off adding extra controls.

Can you fill me up a bit on this?

Would it work if we add the UI control for the GS sidebar but we [keep](https://github.com/WordPress/gutenberg/blob/master/lib/experimental-default-theme.json#L173) the `customRadius` to false? That way we have the support built-in, themes can use it via `theme.json` but users won't see the controls. Same for the other border properties.

Reply:
I have a PR up adding the border radius controls to the global styles sidebar https://github.com/WordPress/gutenberg/pull/28346

> Can you fill me up a bit on this?

Some extra background on past discussions regarding controls for border related support can be found in https://github.com/WordPress/gutenberg/issues/21540#issuecomment-747282075. 

I'm happy to change approach. Whatever we need to get the best result.